front/src/App.js

```diff
@@ -42,11 +42,10 @@ const StickyBottomContainer = styled.div`
 class App extends Component {
   constructor(props) {
     super(props);
-
     this.state = {
       activePage: pageTypes.SETTINGS,
       connectionStatus: connectionTypes.DISCONNECTED,
-      token: "dummy",
+      session: "dummy",
```

Comment:
Why `session`? If this a session token, I'd recommend calling it `sessionToken`.

Reply:
`sessionKey` is used at the backend

Reply:
I actually like session, as in 'the session is {session}'